README.md

```diff
@@ -68,14 +66,34 @@ to something like:
 
 `https://cdn.example.com/assets/Uploads/file.jpg`
 
->>>>>>> rookie-me-feature/cdn_prefix
+You can override the default `/assets/` path by redeclaring the PublicCDNAdapter constructor, with the paramater for the `cdnAssetPath` set to a string of your folder name:
+
+```yml
+---
+Name: silverstripes3-cdn
+Only:
+  envvarset: AWS_PUBLIC_CDN_PREFIX
+After:
+  - "#assetsflysystem"
+  - "#silverstripes3-flysystem"
+---
+SilverStripe\Core\Injector\Injector:
+  SilverStripe\S3\Adapter\PublicAdapter:
+    class: SilverStripe\S3\Adapter\PublicCDNAdapter
+    constructor:
+      s3Client: '%$Aws\S3\S3Client'
+      bucket: "`AWS_BUCKET_NAME`"
+      prefix: "`AWS_PUBLIC_BUCKET_PREFIX`"
+      cdnPrefix: "`AWS_PUBLIC_CDN_PREFIX`"
+      cdnAssetPath: "public" # example of altered path name, which will produce https://cdn.example.com/public/Uploads/file.jpg
```

Comment:
Can we please use a different example path here since `public` folder has a special meaning when dealing with folders and paths in SS4.

```suggestion
      cdnAssetsDir: "cms-assets" # example of a custom assets folder name, which will produce https://cdn.example.com/cms-assets/Uploads/file.jpg
```